code/modules/surgery/surgery.dm

```diff
@@ -78,9 +78,13 @@
 		return FALSE
 
 	// True surgeons (like abductor scientists) need no instructions
-	if(HAS_MIND_TRAIT(user, TRAIT_SURGEON))
+	if(HAS_TRAIT(user, TRAIT_SURGEON))
```

Comment:
Why are you changing this from `HAS_MIND_TRAIT` to `HAS_TRAIT`? The effectively do the same thing (check the macro)

Reply:
Oh, you're right, I had thought it would only check their .mind for the trait but I see it does it for the target and their mind